src/bk-user/bkuser/apis/web/data_source/views.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+"""
+TencentBlueKing is pleased to support the open source community by making 蓝鲸智云-用户管理(Bk-User) available.
+Copyright (C) 2017-2021 THL A29 Limited, a Tencent company. All rights reserved.
+Licensed under the MIT License (the "License"); you may not use this file except in compliance with the License.
+You may obtain a copy of the License at http://opensource.org/licenses/MIT
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+"""
+from drf_yasg.utils import swagger_auto_schema
+from rest_framework import generics, status
+from rest_framework.response import Response
+
+from bkuser.apis.web.data_source.serializers import UserCreateInputSLZ, UserCreateOutputSLZ
+from bkuser.apps.data_source.models import DataSource, DataSourceUser
+from bkuser.biz.data_source_organization import (
+    DataSourceOrganizationHandler,
+    DataSourceUserBaseInfo,
+    DataSourceUserRelationInfo,
+)
+from bkuser.common.error_codes import error_codes
+
+
+class DataSourceUserListCreateApi(generics.ListCreateAPIView):
+    pagination_class = None
+
+    @swagger_auto_schema(
+        operation_description="新建数据源用户",
+        query_serializer=UserCreateInputSLZ(),
+        responses={status.HTTP_201_CREATED: UserCreateOutputSLZ(many=True)},
+        tags=["data_source"],
+    )
+    def post(self, request, *args, **kwargs):
+        slz = UserCreateInputSLZ(data=request.data)
+        slz.is_valid(raise_exception=True)
+        data = slz.validated_data
+        data_source_id = kwargs["id"]
+
+        # 校验数据源是否存在
+        try:
+            data_source = DataSource.objects.get(id=data_source_id)
+        except Exception:
+            raise error_codes.DATA_SOURCE_NOT_EXIST
```

Comment:
data_source = DataSource.objects.filter(id=data_source_id).first()
if not data_source:
    raise error_codes.DATA_SOURCE_NOT_EXIST